internal/pkg/utils/utils.go

```diff
@@ -45,3 +46,44 @@ func FieldName(fa *ssa.FieldAddr) string {
 	}
 	return st.Field(fa.Field).Name()
 }
+
+// DecompoeType returns the path, typename, and indicators for if the Type is Named or an Interface
```

Comment:
```suggestion
// DecomposeType returns the path and name of a Named type.
```

Reply:
I've got to get better about reviewing my docstrings...